### PR TITLE
Add test tags to address book

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it allows testing setting billing and shipping addresses.

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [CU-74rf4c Set as a default billing/shipping address is not correctly displayed](https://app.clickup.com/t/74rf4c)

File: src/@next/components/atoms/DropdownMenu/types.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export interface MenuItem {
   content: React.ReactNode;
+  testId?: string;
```
Comment (krzysztofwolski) [inline src/@next/components/atoms/DropdownMenu/types.ts:3]: Can we keep `testingContext` name in whole project?

Comment (dominik-zeglen) [reply src/@next/components/atoms/DropdownMenu/types.ts:3]: Noted!

**Comment by michalina-graczyk** [comment]
@dominik-zeglen I have merged my changes to the master, please check if there are no conflicts with the newest master.

**Pull request merged**